jsapp/js/account/usage/usage.api.ts

```diff
@@ -63,3 +81,32 @@ export async function getUsageForOrganization() {
 
   return await getUsage(organizations.results?.[0].id);
 }
+
+export async function getAssetUsage(url?: string) {
+  try {
+    const apiUrl = url || ASSET_USAGE_URL;
+    return fetchGet<AssetUsage>(apiUrl, {
+      includeHeaders: true,
+      errorMessageDisplay: t('There was an error fetching asset usage data.'),
+    });
+  } catch (error) {
+    console.error('Error fetching data:', error);
+    throw error;
```

Comment:
Hmm, `fetchGet()` should take care of displaying an error message via a toast, then send the error to our reporting service, which would make this redundant. It _does_ `console.error` (see `jsapp/js/utils.js`), but as you've pointed out through code, it'll only show the `errorMessageDisplay` value, not the error itself. (I think that's my 'oops' originally - I added the `errorMessageDisplay` option and the handling for it.)

So what if, instead of this `catch` block (and the similar one in `jsapp/js/account/usage/usageProjectBreakdown.tsx`), you pass the (optional) real error message to `notify()` in the `ToastOptions`?  Then use it in `handleApiFail()` on line 67.